app/src/services/realitio.ts

```diff
@@ -66,7 +66,7 @@ class RealitioService {
 
     // send the transaction and wait until it's mined
     const transactionObject = await this.contract.askQuestion(...args, {
-      value: ethers.utils.bigNumberify(value),
+      value: '0x0',
```

Comment:
This is already 0. We need to either remove the `value` parameter or to undo these changes. I'm fine with either option (actually, since this was already tested, maybe it's slightly better to just remove the parameter).

Reply:
I will remove the parameter